app/presenters/barclaycard_response_message.rb

```diff
@@ -0,0 +1,22 @@
+module BarclaycardResponseMessage
+  # Information taken from the Barclaycard e-Commerce advanced guide
+  # Technical Integration Guide for e-Commerce v.5.4.0
+
+  AUTHORISATION_NOT_KNOWN = \
+    'Status 52: Authorisation Not Known - A technical problem arose during the
+    authorisation/payment process, giving an unpredicatable result.'.squish.freeze
+
+  PAYMENT_UNCERTAIN = \
+    'Status 92: Payment Uncertain - The customer should not retry the authorisation
+    process since the authorisation/payment might already have been accepted.'.squish.freeze
+
+  def for(status)
+    case status
+    when '52' then AUTHORISATION_NOT_KNOWN
+    when '92' then PAYMENT_UNCERTAIN
+    else "Status Code: #{status}"
+    end
+  end
```

Comment:
I think this entire class could be replaced with Rails's i18n support:

```ruby
I18n.t(status, scope: 'barclaycard.whatever', default: :generic, status: status)
```

With `en.yml` being something like:

```yaml
en:
  barclaycard:
    whatever:
      "52": "Status 52 …"
      "92": "Status 92 …"
      generic: "Status Code: %{status}"
```

This would also have the nice effect of not requiring multi-line string jiggery-pokery in Ruby.